multiplatform/colcon_test/action.yml

```
@@ -29,8 +29,7 @@ inputs:
 
   workspace:
     description: 'Workspace where built has been done'
-    required: false
-    default: ''
+    required: true
```

This may break some other usages of eProsima-CI (in other repos different than Fast-DDS)

---

Internally agreed on set the default value as the platform-specific action ([ubuntu](https://github.com/eProsima/eProsima-CI/blob/5bcd96869682934fbde5239b55bfd44362c57e93/ubuntu/colcon_test/action.yml#L33C14-L33C39), [windows](https://github.com/eProsima/eProsima-CI/blob/5bcd96869682934fbde5239b55bfd44362c57e93/windows/colcon_test/action.yml#L33C14-L33C39))
```suggestion
    '${{ github.workspace }}'
    required: false
```